dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -35,11 +35,20 @@ class LeapHybridSampler(dimod.Sampler):
     Uses parameters set in a configuration file, as environment variables, or
     explicitly as input arguments for selecting and communicating with a hybrid solver.
     For more information, see
-    `D-Wave Cloud Client <https://docs.ocean.dwavesys.com/en/latest/docs_cloud/sdk_index.html>`_.
+    `D-Wave Cloud Client <https://docs.ocean.dwavesys.com/en/stable/docs_cloud/sdk_index.html>`_.
 
     Inherits from :class:`dimod.Sampler`.
 
     Args:
+        solver (dict/str, optional):
```

Comment:
I would like to drop all these args in favor of `**config`, and pointing to cloud-client's docs. The list below is an arbitrary subset anyway, and some are so obscure almost nobody will use them (looking at `proxy` and `connection_close`).

I'm saying this more to start a discussion, we don't have to drop them in this PR. I can do that as part of #317, if we all agree.

Reply:
This was bothering me when I was doing this cleanup, so I'd like to see this.